<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWidget</class>
 <widget class="QWidget" name="ReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="level_lb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="star_btn5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="star_btn3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="toReport_btn">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>263</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>报修</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_tEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>530</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="star_btn1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hurryLevel_lb">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>270</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>着急程度：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="star_btn2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="toComplaint_btn">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>263</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>投诉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="star_btn4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
